docs/sources/installation/debian.md

```diff
@@ -70,99 +75,88 @@ HTTPS.
+### Install .deb package
 
-Start Grafana by running:
+Go to the [download page](https://grafana.com/grafana/download?platform=linux) for the latest download
+links.
 
 ```bash
-sudo service grafana-server start
+sudo wget <tar.gz package url>
+sudo apt-get install -y adduser libfontconfig1
+sudo dpkg -i grafana_<version>_amd64.deb
 ```
 
-This will start the `grafana-server` process as the `grafana` user,
-which was created during the package installation. The default HTTP port
-is `3000` and default user and group is `admin`.
-
-Default login and password `admin`/ `admin`
+## Install from binary .tar file
 
-To configure the Grafana server to start at boot time:
+Download the latest [`.tar.gz` file](https://grafana.com/grafana/download?platform=linux) and extract it. The files extract into a folder named after the Grafana version that you downloaded. This folder contains all files required to run Grafana. There are no init scripts or install scripts in this package.
 
 ```bash
-sudo update-rc.d grafana-server defaults
+sudo wget <tar.gz package url>
+sudo apt-get install -y adduser libfontconfig1
+sudo tar -zxvf <tar.gz package>
 ```
 
-## Start the server (via systemd)
+## 2. Start the server 
+
+This starts the `grafana-server` process as the `grafana` user, which was created during the package installation.
+
+If you installed with the APT repository or .deb package, then you can start the server using systemd or init.d. If you installed a binary .tar file, then you need to execute the binary.
+
+### Start the server with systemd
 
 To start the service using systemd:
 
 ```bash
-systemctl daemon-reload
-systemctl start grafana-server
-systemctl status grafana-server
+sudo systemctl daemon-reload
+sudo systemctl start grafana-server
+sudo systemctl status grafana-server
 ```
 
-Enable the systemd service so that Grafana starts at boot.
+Verify that the service has started:
 
 ```bash
-sudo systemctl enable grafana-server.service
+sudo systemctl status grafana-server
 ```
 
-## Environment file
+Enable the systemd service so that Grafana starts at boot:
 
-The systemd service file and init.d script both use the file located at
-`/etc/default/grafana-server` for environment variables used when
-starting the back-end. Here you can override log directory, data
-directory and other variables.
+```bash
+sudo systemctl enable grafana-server.service
+```
 
-### Logging
+### Start the server with init.d
 
-By default Grafana will log to `/var/log/grafana`
+Start Grafana by running:
 
-### Database
+```bash
+sudo service grafana-server start
+```
 
-The default configuration specifies a sqlite3 database located at
-`/var/lib/grafana/grafana.db`. Please backup this database before
-upgrades. You can also use MySQL or Postgres as the Grafana database, as detailed on [the configuration page]({{< relref "configuration.md#database" >}}).
+Verify that the service has started:
 
-## Configuration
+```bash
+sudo NEED CONTENT HERE
```

Comment:
It would be great if we could tell them how to verify the service has started for each start method. If such a command exists, please add it.

Reply:
sudo service grafana-server status